geoengine/workflow.py

```diff
@@ -64,6 +64,7 @@ class Workflow:
 
     __workflow_id: WorkflowId
     __result_descriptor: ResultDescriptor
+    __timeout: int = 10
```

Comment:
Some methods in `workflows.py` use this variable and others have it in their params with a default value.
I would prefer the second method here. Moreover, maybe change 10 to 60 seconds. We just don't want to wait forever.

Reply:
[12478ac](https://github.com/geo-engine/geoengine-python/pull/71/commits/12478ac297e420b824d9393ef19c88dcd97b5e3b)

Removed the field and made it a parameter for all methods with 60s default.